### ISSUE TITLE
Thanks for possible help. I failed to get the result in the paper with public weights

### ISSUE THREAD

**Issue by Jingling1** [gloriatao/LAST#1]
Thanks a lot for your possible help!

I use two NVIDIA TITAN V GPUs to run inference_stage1.py with your public stage1-light.pth
I don't generate all .npy files to save storage
and I use the frame index of 1.png, 26.png, and 25×n+1 etc for testing with the label of frame 0, 25, and 25×n. I have checked with your preprocess_video2npy_multiprocessing.py and torch.utils.data.dataset, I'm nearly 100% sure that there is no error.

I get the test VFE result on video41 as
acc: 0.40947164948453607 pr: 0.367242801202603 re: 0.40794866382746264 jacc: 0.22119418477527075 mAP: 0.6955197179487798
that is too terrible

all results are:
accuracy: [0.40947165 0.52922165 0.38129496 0.4862532  0.45631641 0.46916566
        0.30752212 0.43978202 0.45992823 0.41826484 0.4278438  0.44331469
        0.43666261 0.42889391 0.50192678 0.40653951 0.42233194 0.4337671
        0.51384909 0.4086064  0.43514739 0.45548451 0.45078626 0.45977491
        0.53628602 0.47068493 0.43840272 0.45970603 0.52534965 0.46861925
        0.39586645 0.40843257 0.49299926 0.42507645 0.38565489 0.42490566
        0.47463044 0.40810811 0.51010249 0.48463768] precision: [0.3672428  0.55949213 0.3659321  0.40958154 0.43469805 0.54054294
        0.38053907 0.5756021  0.54232261 0.55911757 0.49309918 0.67008249
        0.39707009 0.51757807 0.50881025 0.44774628 0.46861636 0.38184485
        0.54702826 0.27842354 0.35029102 0.61412046 0.4951658  0.44147422
        0.60747463 0.5496874  0.39546083 0.56315495 0.48026881 0.50436302
        0.37418421 0.40526827 0.49560044 0.41520113 0.38622664 0.37115024
        0.4435746  0.47639307 0.51309166 0.35176913] recall: [0.40794866 0.62260763 0.3899639  0.4320977  0.40627225 0.52525708
        0.27026372 0.47360543 0.467428   0.41693542 0.49986827 0.53702389
        0.52660183 0.37662229 0.39067293 0.44420244 0.46249449 0.48004331
        0.55904021 0.4322109  0.43716617 0.60471176 0.64490555 0.57679797
        0.61493874 0.51569028 0.44660507 0.57637549 0.53836399 0.49374968
        0.40933515 0.42380575 0.52041722 0.42084094 0.47137356 0.37767818
        0.39378276 0.4148499  0.62539989 0.426009  ] jaccard: [0.22119418 0.34503292 0.19315962 0.23450754 0.25369294 0.36450081
        0.18110287 0.33971229 0.30421907 0.32573059 0.30759566 0.35645214
        0.2195398  0.27021339 0.28803336 0.27726205 0.27176297 0.22289473
        0.3509587  0.17518647 0.20966681 0.28481365 0.30522024 0.27152645
        0.41737668 0.33177953 0.24105786 0.31575981 0.29556087 0.29950686
        0.18770322 0.23999821 0.29648037 0.25899573 0.22904317 0.20850622
        0.2711442  0.27240865 0.34801377 0.26543053] mAPs: [0.69551972 0.60733148 0.45473768        nan 0.38988599 0.59940223
        0.47693351        nan 0.47206545        nan 0.56452159 0.53400281
        0.68462604 0.40484598        nan        nan 0.48312592 0.39461162
        0.46749814 0.44778902        nan 0.46615694 0.48749281 0.65154548
        0.4760372  0.54390061        nan 0.50790698        nan        nan
                nan 0.50224168 0.6068046  0.59783774        nan        nan
        0.47310924        nan 0.55137322        nan]
        -average accuracy: 0.44729030599192166 -std: 0.04518993573848753
        -average precision: 0.4669822700819582 -std: 0.08653083303949799
        -average recall: 0.47634893553311103 -std: 0.08255977812979837
        -average JACC score: 0.27631862400194046 -std: 0.055802137058991404
        -average mAP: nan -std: nan

**Comment by Jingling1** [comment]
![Uploading 捕获.PNG…]()

**Comment by Jingling1** [comment]
while the VFE metric results in the paper are
accuracy: 83.32%
precision: 73.72%
recall: 79.03%
mAP: 90.88%

**Comment by Jingling1** [comment]
Is the problem with the public weights? Or could you please help me to point out the problem
